torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp

```diff
@@ -64,13 +64,6 @@ static std::vector<std::string> TORCH_NCCL_ASYNC_ERROR_HANDLING = {
 static std::vector<std::string> TORCH_NCCL_DUMP_ON_TIMEOUT = {
     "TORCH_NCCL_DUMP_ON_TIMEOUT"};
 
-// TODO: remove this change after a safe rollout.
-// Control whether we sleep after an exception is thrown.
-// This change is temporary and is used to safely remove the current sleep that
-// exists after an exception is thrown.
-static std::vector<std::string> TORCH_NCCL_SLEEP_AFTER_EXCEPTION = {
-    "TORCH_NCCL_SLEEP_AFTER_EXCEPTION"};
```

Comment:
also I would ensure this won't affect llama. Does llama job use JK knobs?

Reply:
No, llama doesn't use JKs. However, they do have DUMP_ON_TIMEOUT enabled. So, with this change, they will wait, at most one minute, on timed out jobs so that FR dumps can be written out.